### PR TITLE
DAOS-2972 test: Validate SCM configuration

### PR DESCRIPTION
**Pull request by amjustin13**
As part of this patch two new test files have been added:
daos_scm_config.py
daos_scm_config.py

Test cases:
- Configuring devices that have already been configured and are in use by DAOS

### CODE REVIEW

**Comment by amjustin13** [comment]
I am having issues with the second test case as I am trying to use the ipmctl-inject-error command to inject a mock error. I reached out to someone in AEP architecture team to get more info on the issue, so I removed that test case until that issue is resolved:
- Configuring devices that are not in a usable state

File: src/tests/ftest/control/daos_scm_config.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        self.add_container(self.pool)
+
+        # now open the container and write some data
+        self.container.open()
+        data_w = "Ehrm... Testing... Testing"
+        self.write_data(data_w)
+
+        # Run storage prepare
+        if self.server_managers[-1].manager.job.using_dcpm:
+            # Storage prepare should return error
```
Comment (shimizukko) [inline src/tests/ftest/control/daos_scm_config.py:80]: I'm confused about this comment. Is Line 84 supposed to return error and raise an exception? If that's the expectation, why do we have self.fail in Line 86?

Comment (amjustin13) [reply src/tests/ftest/control/daos_scm_config.py:80]: Good catch, I should remove that comment. It's not relevant to the test case, I must've copy pasted it and did not modify it.

**Comment by amjustin13** [comment]
CI test failure due to:
- DAOS rebuild tests: https://jira.hpdd.intel.com/browse/DAOS-4623

**Pull request merged**